src/Tools/GenerateProps/targetgroups.props

```diff
@@ -184,5 +184,8 @@
       <!-- this targetgroup is not considered compatible with any other targetgroup so it should only
            build when specified directly or BuildAllConfigurations is set to True.  -->
     </TargetGroups>
+    <TargetGroups Include="mono">
+      <NuGetTargetMoniker>.NETFramework,Version=v4.6.3</NuGetTargetMoniker>
```

Comment:
You are going to want to change this to `.NETCoreApp,Version=v2.0` and add some compatible with targets to "netcoreapp" otherwise you will have to add a mono configuration for every library in the closure. System.Private.Uri is special in that it only depends on S.P.CoreLib which you are getting because of your runtime update, but as soon as you need other references you are going to need to update this.